test/integration/garbagecollector/garbage_collector_test.go

```diff
@@ -443,7 +443,7 @@ func setupRCsPods(t *testing.T, gc *garbagecollector.GarbageCollector, clientSet
 	rc.ObjectMeta.Finalizers = initialFinalizers
 	rc, err := rcClient.Create(context.TODO(), rc, metav1.CreateOptions{})
 	if err != nil {
-		t.Fatalf("Failed to create replication controller: %v", err)
+		errs <- err
```

Comment:
```suggestion
		errs <- fmt.Errorf("Failed to create replication controller: %v", err)
```
it might be better to perserve the original error message, like